src/moduleRoutes.js

```diff
@@ -50,7 +50,7 @@ function getModuleRoutes(stripes) {
                       </div>
                     </AddContext>
                   </StripesContext.Provider>);
-                }
+              }
```

Comment:
Was this intentional? It doesn't look like it's indented properly with the `}` below

Reply:
Fixed